<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>760</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>228</green>
        <blue>254</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>228</green>
        <blue>254</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>228</green>
        <blue>254</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>228</green>
        <blue>254</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>87</y>
     <width>285</width>
     <height>628</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
	background-color: rgb(64, 66, 226)
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>285</width>
      <height>74</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(64, 66, 226);
	color: rgb(255, 255, 255);
}
QPushButton::hover
{
	background-color: rgb(0, 170, 0);
}</string>
    </property>
    <property name="text">
     <string>STUDENT INFO</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>285</width>
      <height>74</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(64, 66, 226);
	color: rgb(255, 255, 255);
}
QPushButton::hover
{
	background-color: rgb(0, 170, 0);
}</string>
    </property>
    <property name="text">
     <string>TRAIN DATA</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>285</width>
      <height>74</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Black</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(64, 66, 226);
	color: rgb(255, 255, 255);
}
QPushButton::hover
{
	background-color: rgb(0, 170, 0);
}</string>
    </property>
    <property name="text">
     <string>PHOTO</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>285</width>
      <height>74</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Black</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(64, 66, 226);
	color: rgb(255, 255, 255);
}
QPushButton::hover
{
	background-color: rgb(0, 170, 0);
}</string>
    </property>
    <property name="text">
     <string>RECOGNIZE FACE</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>285</width>
      <height>74</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Black</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(64, 66, 226);
	color: rgb(255, 255, 255);
}
QPushButton::hover
{
	background-color: rgb(0, 170, 0);
}</string>
    </property>
    <property name="text">
     <string>NOTIFY GUARDIAN</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>285</width>
      <height>74</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Black</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(64, 66, 226);
	color: rgb(255, 255, 255);
}
QPushButton::hover
{
	background-color: rgb(0, 170, 0);
}</string>
    </property>
    <property name="text">
     <string>ATTENDANCE REPORT</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>285</width>
      <height>84</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Black</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(64, 66, 226);
	color: rgb(255, 255, 255);
}
QPushButton::hover
{
	background-color:rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>LOGOUT</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>25</y>
     <width>921</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Black</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(93, 136, 206);</string>
   </property>
   <property name="text">
    <string>FACE RECOGNITION ATTENDANCE SYSTEM</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ICON</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>993</width>
     <height>620</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 228, 254);</string>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="train_data_page">
    <widget class="QPushButton" name="train_pushbutton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>350</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: rgb(64, 66, 226);
	color: rgb(255, 255, 255);
}

QPushButton::hover
{
	background-color:rgb(0, 170, 0);
}</string>
     </property>
     <property name="text">
      <string>TRAIN ALGORITHM</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="photo_page">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>290</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SEARCH</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>295</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>360</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: rgb(64, 66, 226);
	color: rgb(255, 255, 255)
}
QPushButton::hover
{
	background-color: rgb(0, 170, 0);
}</string>
     </property>
     <property name="text">
      <string>OPEN FOLDER</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="student_info_page">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>230</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SUDENT PAGE INFO</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="notify_guardian_page">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Black</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SEARCH</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="to_line_edit">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>60</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>To: (Guardian Email)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="subject_line_edit">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>110</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Subject</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="mail_content_edit">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>160</y>
       <width>281</width>
       <height>351</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="placeholderText">
      <string>Mail Content</string>
     </property>
    </widget>
    <widget class="QPushButton" name="send_mail_button">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>530</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Black</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: rgb(64, 66, 226);
	color: rgb(255, 255, 255)
}
QPushButton::hover
{
	background-color: rgb(0, 170, 0);
}</string>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="attendance_report_page">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>260</y>
       <width>291</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ATTENDANCE REPORT</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="recognize_face_page">
    <widget class="QPushButton" name="recognize_face_button">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>310</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: rgb(64, 66, 226);
	color: rgb(255, 255, 255);
}

QPushButton::hover
{
	background-color:rgb(0, 170, 0);
}</string>
     </property>
     <property name="text">
      <string>RECOGNIZE FACES</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
